### ISSUE TITLE
Use original request if the reroute fails

### ISSUE THREAD

**Issue by jonathandsouza** [seriema/angular-apimock#66]
Hi ,

Is there a way to configure angular-apimock to use the original request if the reroute fails. I believe in the current implementation its possible to do the opposite (i.e try the original request, if it fails then it tries to recover with a reroute) by setting apimock=auto. 

Thanks & Regards

**Comment by seriema** [comment]
You're right, it's possible to do the opposite of what you want. I'll consider it, but it might take a while. I'd gladly appreciate a PR if you want to contribute.
